app/base_scalers.py

```diff
@@ -63,6 +63,10 @@ def __init__(self, app_name, min_instances, max_instances):
         self.last_db_error = datetime.min
 
     def _init_db_uri(self):
+        if "SQLALCHEMY_DATABASE_URI" in os.environ:
+            self.db_uri = os.environ["SQLALCHEMY_DATABASE_URI"]
+            self.db_uri = self.db_uri.replace("postgresql://", "postgres://")
```

Comment:
No action required but this really got me confused...

I looked at the API and saw that we were replacing `postgres` with `postgresql`. The opposite! https://github.com/alphagov/notifications-api/blob/main/app/cloudfoundry_config.py#L11

I did some digging and it turns out this is OK.

It looks like

For the API, the PaaS provides a uri which starts `postgres` and we want to force that to become `postgresql`. In our credentials file where we are setting our custom uri, it already starts with `postgresql`.

However for the autoscaler, the PaaS provides a uri which starts `postgres` and we keep that as is, we don't change it to `postgresql`. This means that to match the schema, we need to change our credential from `postgresql` to `postgres`, which is what you've done here.

Essentially, I was surprised to see us using `postgres` as the schema here but `postgresql` in the API and that threw me off when reviewing this PR...

Reply:
With the substitutions the way they are, the user could provide postgres:// or postgresql:// and both would work.